### ISSUE TITLE
Add. 17,251 | https://bl.syriac.uk/ms/1043

### ISSUE THREAD

**Issue by GiuliaSecli** [srophe/britishLibrary-data#1644]
In Wright's pdf there are three small symbols between two proper names in Greek, how do I encode the symbols?

![image](https://github.com/srophe/britishLibrary-data/assets/171020243/2d8cd1ef-459e-42d5-8d03-baf4e7b3f984)

**Comment by davidamichelson** [comment]
Unfortunately there is no way at present to represent these symbols (you can check here but I did not see them https://www.unicode.org/charts/PDF/U0370.pdf).

In this case please just leave this issue open (and add to the subject line the following: PDF contains characters or images which cannot be displayed in XML

**Comment by davidamichelson** [comment]
Additional note, @wlpotter how will we handle cases like this where it is not clear if the text is Greek or Coptic?

**Comment by GiuliaSecli** [comment]
I think it could be Greek uncial script. 
The problem now is marking the delta (or dalda) upside down. 
Regarding the third symbol, it's probably a qoppa, but I'm not sure if it indicates a numeral. 
The characters of the whole line aren't visible in the XML file or preview.

![image](https://github.com/srophe/britishLibrary-data/assets/171020243/c4838f7b-592b-425e-9821-33d63652d8b4)

I don't know how to solve this: do I add a comment, save as written and commit?

**Comment by GiuliaSecli** [comment]
I see the characters in GitHub:

![image](https://github.com/srophe/britishLibrary-data/assets/171020243/9c234847-c2bd-4576-ac89-9daa57f657a7)

**Comment by davidamichelson** [comment]
Thanks @GiuliaSecli you can just leave this issue open and we will check over the Greek transcription and any missing characters. I think that character that looks like an inverted delta may be a upsilon (for Σευῆρε).

In these cases where not all of the characters can be encoded, please open a GitHub issue, apply the label "cannot display", and include this message in the title of your issue: “PDF contains characters or images which cannot be displayed in XML.” Will Potter and David Michelson will handle these issues.

**Comment by GiuliaSecli** [comment]
Thanks a lot, Dr. Michelson. 
I agree it's an upsilon, but I've never seen it written like that. 
Just to be sure, do I still have to commit the other changes I made during the revision?

**Comment by davidamichelson** [comment]
Yes please do make a commit and push, but then leave this issue open so we
know there are still display issues.
